### DISCUSSION TITLE
ESP32-S3 Parallel I80 LCD (a.k.a Intel 8080 parallel LCD)

### DISCUSSION THREAD

**Discussion by josef2600** [Bodmer/TFT_eSPI#3300] [New processor/display support or capability request] [answered]
HI,
i have searched, but i didn't find any discussion related to this one.
espressif and its datasheet say ESP32-S3 has a dedicated internal driver for connecting a LCD directly to ESP32-S3 without any driver chip.
i checked the readme and couldn't fined anything related to it. is it possible for any one that add parallel lcd code to this driver?
like this link:
https://docs.espressif.com/projects/esp-idf/en/stable/esp32s3/api-reference/peripherals/lcd.html

they say their library supports : I80 LCD (a.k.a Intel 8080 parallel LCD), RGB/SRGB LCD . and in their code i have seen they do have a 8-8-8 or 24 bit driver in code, but in the hardware , i only see 16 bit. 
so, why parallel interface without driver chip ? because:
1. sizes of the LCD's. usually if the size goes up, like 5" or more, ether it doesn't exist in general stores, or they are extremely expensive.
2. most of IPS LCD's are without driver chip. they only support direct connection and don't have any driver or supporting driver.
3. resolution. when we use direct parallel connection, and using PSRAM, in theory(!) we can support 1080p. obviously it is not going to work, because of the speed. but it can work with 1024 * 720 with a good speed. now it can even support 15" lcd!
4. and the speed of communication is extremely fast. because there is no external driver, LCD can run at the full speed of the LCD. usually it is 40mhz. no driver has that speed. just to be clear, the internal driver is directly connected to CPU buss. and it does generate all LCD clock's and communications like: LCD_PCLK LCD_DC LCD_V_SYNC LCD_H_SYNC
5. it will be the cheapest driver LCD.
6. it works with Arduino IDE.

thank you.

**Comment by FerrariForever95** [comment, marked as answer]
i am wokring on this too , i need time lets connect i am also working on esp32 s3 with ili9488 8 bit bud i8080 interface , once i get the drivers ready , i will letyou know

**Reply by josef2600** [reply]
i cant believe it has been 2 years! 
thanks.

**Reply by FerrariForever95** [reply]
I have working driver[]()![IMG_20260717_001659.jpg](https://github.com/user-attachments/assets/4d7bd0f5-03b2-4d3a-97a1-c127ad66be51)
